processor/otel/consumer.go

```diff
@@ -152,7 +152,7 @@ func (c *Consumer) convertSpan(
 	var span *model.Span
 
 	name := otelSpan.Name()
-	if root || otelSpan.Kind() == pdata.SpanKindSERVER {
+	if root || otelSpan.Kind() == pdata.SpanKindSERVER || otelSpan.Kind() == pdata.SpanKindCONSUMER {
```

Comment:
probably worth adding a comment here, and an issue for handling active/passive consumption differently

Reply:
Added a [comment](https://github.com/elastic/apm-server/pull/5334/commits/98d4f8503979831fce8709bb926050018df0b5f5) and an issue https://github.com/elastic/apm-server/issues/5373